pkg/apiserver/admission/admissiontimeout/decorator.go

```diff
@@ -6,17 +6,13 @@ import (
 	"k8s.io/apiserver/pkg/admission"
 )
 
-// AdmissionTimeout provides a decorator that will fail an admission plugin after a certain amount of time
-//
-// DEPRECATED: use the context of the admission handler instead.
-type AdmissionTimeout struct {
```

Comment:
why not keep this backwards compatible? 
otherwise , revendoring will need to change code in the consumers

Reply:
I'm not familiar with this code, whatever you and @tkashem decide LGTM